chat-plugins/info.js

```diff
@@ -10,8 +10,7 @@
  * @license MIT license
  */
 
-var commands = exports.commands = {
```

Comment:
It's so you can refer to `commands` as a local variable. In practice, we never do this (heck, the one place we could have, I used `CommandParser.commands` to be safe).

So I _guess_ we could have deleted this, although probably in a separate explicit commit and see if other files do it so they can also be changed.